abi/abi.go

```diff
@@ -1004,17 +1012,17 @@ func SevProductFromCpuid1Eax(eax uint32) *pb.SevProduct {
 	// Product information specified by processor programming reference publications.
 	switch family {
 	case zen3zen4Family:
-		switch model {
+		switch extendedModel {
 		case milanModel:
 			productName = pb.SevProduct_SEV_PRODUCT_MILAN
-		case genoaModel:
+		case genoaModel, sienaModel:
```

Comment:
The Siena model in the zen4 family should still have the Siena product name.

Reply:
I don't think we should try to claim support without running on hardware. I also don't have access to a siena machine.